### PR TITLE
chore: add ga to web-api

### PR DESCRIPTION
**Pull request by mumin-khan**

### CODE REVIEW

File: cryptlex/cryptlex-enterprise/values.yaml [hunk 1]
```diff
@@ -250,6 +250,8 @@ webApi:
   # Postmark
   postmark:
     accountToken: ""
+  googleAnalytics:
+    key: "G-XXXXXXX"
```
Comment (adnan-kamili) [inline cryptlex/cryptlex-enterprise/values.yaml]: @mumin-khan set deafult value to an empty string, sample value cam be added as comment. We can also club google related settings under google (recaptacha and analytics)

File: cryptlex/cryptlex-enterprise/values.yaml [hunk 2]
```diff
@@ -250,6 +250,8 @@ webApi:
   # Postmark
   postmark:
     accountToken: ""
+  googleAnalytics:
+    key: ""  # G-XXXXXXX
```
Comment (adnan-kamili) [inline cryptlex/cryptlex-enterprise/values.yaml]: keep comments above the property and move this to google section

Comment (mumin-khan) [reply cryptlex/cryptlex-enterprise/values.yaml]: @adnan-kamili we use recaptcha and analyticskey only in web-api, so is making a seperate google section necessary at this point ?

Comment (adnan-kamili) [reply cryptlex/cryptlex-enterprise/values.yaml]: Third party config is usually grouped unded third party provider name so:
```
 # Google
  google:
    recaptcha:
       secretKey: ""
    analytics:
       key: ""
```

File: cryptlex/cryptlex-enterprise/values.yaml [hunk 3]
```diff
@@ -255,3 +252,10 @@ sso:
   google:
     # Refer to following for enabling Google SSO: https://docs.cryptlex.com/user-management/google-sso
     clientId: ""
+ # Google
+  google:
```
Comment (adnan-kamili) [inline cryptlex/cryptlex-enterprise/values.yaml:251]: please ensure it is a separate section not child of sso, infact we can remove sso section and bring it under google

File: cryptlex/cryptlex-enterprise/values.yaml [hunk 4]
```diff
@@ -250,8 +247,13 @@ webApi:
   # Postmark
   postmark:
     accountToken: ""
-# SSO config
-sso:
-  google:
+# Google
+google:
```
Comment (adnan-kamili) [inline cryptlex/cryptlex-enterprise/values.yaml]: @mumin-khan please be carefuf in yaml, incorrect spacing changes the meaning of the property. google is wrongly spaced

Comment (mumin-khan) [reply cryptlex/cryptlex-enterprise/values.yaml]: @adnan-kamili I don't think so.

Comment (mumin-khan) [reply cryptlex/cryptlex-enterprise/values.yaml]: image
Although I agree I have made quite a few  silly mistakes from the past few days.

File: cryptlex/cryptlex-enterprise/templates/config-maps/web-portals-configmap.yaml [hunk 5]
```diff
@@ -18,4 +18,4 @@ data:
   WEB_API_BASE_URL: http://{{ .Values.ingress.hosts.webApiHost }}
   RELEASE_SERVER_BASE_URL: http://{{ .Values.ingress.hosts.releaseServerHost }}
   {{ end }}
-  GOOGLE_CLIENT_ID: {{ .Values.sso.google.clientId | quote }}
+  GOOGLE_CLIENT_ID: {{ .Values.google.sso.clientId | quote }}
```
Comment (mumin-khan) [inline cryptlex/cryptlex-enterprise/templates/config-maps/web-portals-configmap.yaml]: @adnan-kamili this is going to be removed in the other pr.

**Review by adnan-kamili — APPROVED** [review]

**Pull request merged**